clients/form-builder/index.js

```diff
@@ -124,19 +124,21 @@ const HandleSave = ({ jQuery, formId }) => {
 		} else {
 			data_fields.config.processors = {};
 		}
+
 		
-		//Clear all assignments of fields to conditionals
-		if (data_fields.config.hasOwnProperty("fields")) {
-			Object.keys(data_fields.config.fields).forEach((fieldId) => {
-				if (data_fields.config.fields.hasOwnProperty(fieldId)) {
-					data_fields.config.fields[fieldId].conditions = {
-						type: "",
-					};
-				}
-			});
-		}
+		if( typeof fieldID !== "undefined" ){
```

Comment:
@New0 I don't see `fieldID` defined in the scope of HandleSave function.

Reply:
@Shelob9 you're right, I'm so confused. I am able to save Form settings with conditionals using that condition and it also fixes the issue of the connected forms... I might have misread the l36 thinking it was globally defining a fieldID.

Reply:
I see what you mean. While I was looking at L36 I saw:  `const { formFields } = React.useContext(FormFieldsContext);` Maybe we could do that in `handleSave` and check if formFields is undefined or is empty array -- what it should be.